temporalnexus/operation.go

```diff
@@ -94,6 +93,28 @@ func NewSyncOperation[I any, O any](
 	}
 }
 
+// SignalWorkflowInput is the input to a NewSignalWorkflowOperation.
+type SignalWorkflowInput = internal.ClientSignalWorkflowInput
```

Comment:
Don't reuse the type from the interceptors, you want a type that contains the workflow ID, optional run ID and signal name.